src/drive/web/modules/navigation/Breadcrumb/components/DesktopBreadcrumb/DesktopBreadcrumb.jsx

```diff
@@ -0,0 +1,112 @@
+import React, { useEffect, useMemo, useState } from 'react'
+import classNames from 'classnames'
+import { default as BreadcrumbMui } from '@material-ui/core/Breadcrumbs/Breadcrumbs'
+import Icon from 'cozy-ui/transpiled/react/Icon'
+import RightIcon from 'cozy-ui/transpiled/react/Icons/Right'
+
+import styles from 'drive/web/modules/navigation/Breadcrumb/breadcrumb.styl'
+import { useI18n } from 'cozy-ui/transpiled/react/I18n'
+import ActionMenu, { ActionMenuItem } from 'cozy-ui/transpiled/react/ActionMenu'
+
+const DesktopBreadcrumb = ({ onBreadcrumbClick, path }) => {
+  const { t } = useI18n()
+
+  const expandText = useMemo(() => t('breadcrumb.label'), [t])
+  const [dropdownTrigger, setDropdownTrigger] = useState(
+    document.querySelector(`[aria-label="${expandText}"]`)
+  )
+  const anchorElRef = useMemo(() => ({ current: dropdownTrigger }), [
+    dropdownTrigger
+  ])
+  const [menuDisplayed, setMenuDisplayed] = useState(false)
+
+  const handleDropdownTriggerClick = e => {
+    e.stopPropagation()
+    setMenuDisplayed(true)
+  }
+
+  useEffect(() => {
+    setMenuDisplayed(false)
+    setDropdownTrigger(document.querySelector(`[aria-label="${expandText}"]`))
+  }, [path]) // eslint-disable-line react-hooks/exhaustive-deps
+
+  useEffect(() => {
+    const trigger = anchorElRef.current
+    if (trigger) {
+      trigger.addEventListener('click', handleDropdownTriggerClick)
+      return () => {
+        setMenuDisplayed(false)
+        trigger.removeEventListener('click', handleDropdownTriggerClick)
+      }
+    }
+  }, [anchorElRef.current]) // eslint-disable-line react-hooks/exhaustive-deps
```

Comment:
why don't you pass `setMenuDisplayed` ?

Reply:
not necessary: https://reactjs.org/docs/hooks-reference.html#usestate



    Note

    React guarantees that setState function identity is stable and won’t change on re-renders. This is why it’s safe to omit from the useEffect or useCallback dependency list.